Clarify dbt docs generation guidance (#9517)
fishtown-analytics/docs.getdbt.com: website/docs/docs/build/documentation.md

Generate documentation for your project by following these steps:

1. Run the `dbt docs generate` [command](/reference/commands/cmd-docs#dbt-docs-generate) to compile relevant information about your dbt project and warehouse into `manifest.json` and `catalog.json` files, respectively. 

>    Before generating docs, save any updates to model, source, and column descriptions in your YAML files. If your workflow uses selectors or exclusions, generate documentation from the same project context you use for development so the generated `manifest.json` and `catalog.json` reflect the resources dbt parses for that run.

Reviewer: ```suggestion
Before generating docs, save your YAML description updates for models, sources, and columns. Use the same project context you use for development, including any selectors or exclusions, so `manifest.json` and `catalog.json` match the resources dbt parses for that run.
```
